netci.groovy

```diff
@@ -109,7 +109,7 @@ def project = 'dotnet/wcf'
 // **************************
 
 // Define build string
-def codeCoverageBuildString = '''build.cmd /p:ShouldCreatePackage=false /p:ShouldGenerateNuSpec=false /p:OSGroup=Windows_NT /p:Configuration=Windows_NT_Debug /p:Coverage=true /p:WithCategories=\"\\\"InnerLoop;OuterLoop\\\"\"'''
```

Comment:
I got a little overzealous in escaping the commands - in Windows it's not necessary to have this escaping, hence this change is needed as well.